### PR TITLE
refactor: setup needed messaging

### PR DESCRIPTION
**Pull request by frosso**
# Changes proposed in this Pull Request:

Adjusting the UI of the "Setup needed" messaging from text to a pill

Before:
Screen Shot 2021-09-15 at 12 04 22 PM

After:
Screen Shot 2021-09-15 at 12 02 07 PM

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` feature enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- UI should be updated

I didn't add any tests because everything is hardcoded, for now.

### CODE REVIEW

**Comment by tommyshellberg** [comment]
This is working and looks nice - I like that it draws more attention to the fact that the method isn't yet activated. 👍 
However, when we only show `Pending activation`, are we telling users they just need to wait?
The previous messaging was that more information was needed and with this iteration I think our users will believe they just need to wait.

IMO it wouldn't be bad to have the pill as a visual cue and keep the original messaging at the bottom along with a call to action(a link to their Stripe account to update their info?)

**Comment by frosso** [comment]
@tommyshellberg the "pending activation" and "please provide more information" messages will be represented by 2 different states in the UI. A link will be added to help manage the payment methods within Stripe

File: client/settings/general-settings-section/payment-method-description.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
  */
 import React from 'react';
 import styled from '@emotion/styled';
+import PaymentMethodSetupHelp from './payment-method-setup-help';
```
Comment (tommyshellberg) [inline client/settings/general-settings-section/payment-method-description.js:10]: This should be prefaced with an Internal dependencies docblock ^

**Review by tommyshellberg — APPROVED** [review]
LGTM 💯

File: client/settings/general-settings-section/payment-method-description.js (hunk 1 above)
Comment (frosso) [reply client/settings/general-settings-section/payment-method-description.js:10]: Ah thanks, fixed in a08b9b2b75ecd650d1f12f1a81e8501ea3b9fbb8 - FYI we're removing that rule in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1854#discussion_r709661064

Comment (tommyshellberg) [reply client/settings/general-settings-section/payment-method-description.js:10]: 🥳 I'm not a fan myself

**Pull request merged**